backend/app/api/diagnosis.py

```
@@ -0,0 +1,252 @@
+            "crop": h.crop,
+            "region": h.region,
+            "analysis_result": h.analysis_result,
+            "image_url": h.image_url,
+            "created_at": h.created_at.isoformat()
+        } for h in histories
+    ]
+
+@router.post("/history")
+async def create_diagnosis_history(
+    payload: dict,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)